lib/output.ts

```diff
@@ -6,15 +8,17 @@ export function mismatchingVersionsToOutputLines(
 ): string[] {
   return mismatchingDependencyVersions.map(
     (obj) =>
-      `${obj.dependency} has more than one version: ${obj.versions
+      `${chalk.bold(
+        obj.dependency
+      )} has more than one version:\n  ${obj.versions
         .sort((a, b) => compareRanges(a.version, b.version))
         .map(
           (versionObj) =>
             `${versionObj.version} (${versionObj.count} ${pluralizeUsage(
               versionObj.count
             )})`
         )
-        .join(', ')}`
+        .join('\n  ')}`
```

Comment:
We should only join with a newline `\n` here and no spaces. Inside `.map(` above, we can add a tab character `\t` to indent each line.